Makefile

```diff
@@ -263,10 +263,19 @@ controller-gen: ## Download controller-gen locally if necessary.
 	@test '$(shell $(CONTROLLER_GEN) --version)' = 'Version: $(controller_gen_version)' ||\
 	$(call go-get-tool,sigs.k8s.io/controller-tools/cmd/controller-gen@$(controller_gen_version))
 
+.PHONY: kustomize
 KUSTOMIZE = $(shell pwd)/bin/kustomize
+KUSTOMIZE_VERSION = v4.5.7
 kustomize: ## Download kustomize locally if necessary.
-	@test -f $(KUSTOMIZE) ||\
-	$(call go-get-tool,sigs.k8s.io/kustomize/kustomize/v4@v4.5.7)
```

Comment:
We used here a make function which makes the code much nicer - now we seem to reinvent it. Can we use the function and change only the check for the version?

Reply:
This approach was using go get. Pro of go get approach is that we get it in one command. Con is that it is slower. Let me check what to do.